ivy.xml

```diff
@@ -21,7 +21,7 @@
   <dependency org="com.yahoo.platform.yui" name="yuicompressor" rev="2.4.2-zimbra"/>
   <dependency org="rhino" name="js" rev="1.6R7"/>
   <dependency org="zimbra" name="zm-soap" rev="latest.integration"/>
-  <dependency org="zimbra" name="zm-ajax" rev="latest.integration"/>
+  <dependency org="zimbra" name="zm-admin-ajax" rev="latest.integration"/>
```

Comment:
just to make sure, we will not be merging these changes till build script changes are not completed, otherwise build will be broken